lib/__tests__/Importer-test.js

```diff
@@ -104,7 +104,7 @@ describe('Importer', () => {
       it('displays a message', () => {
         subject();
         expect(result.messages[0]).toMatch(
-          RegExp(`No JS module to import for variable \`${word}\``));
+          RegExp(`No JS module to import for \`${word}\``));
```

Comment:
Yeah, we should. I'll update this and other places. I think we did it in the past because we used to show timings.